src/core/geometry/qgsrectangle.h

```diff
@@ -54,6 +54,13 @@ class CORE_EXPORT QgsRectangle
     ~QgsRectangle() = default;
 
     /**
+    * Creates a new rectangle from a WKT string.
+    * The WKT must contain only 4 vertices, representing a rectangle.
```

Comment:
Shouldn't this be 5 vertices?

Reply:
Better also mention here that only rectangles aligned with the X/y axis are supported.